jalbe0076/whats-cooking
# Refactor/code norms

### What is the feature?
- No new features added, refactored code to remove unused variables/exports and to remove whitespace and comments
- Ran `npm run build` to create a built directory

### How you implemented the solution?
- N/A

### Does it impact any other area of the project?
- N/A

### Any changes in the UI (Screenshots)?
- None

### How to test the feature (A test scenario or any new setup)?
- Pull the changes to your local pc and run `npm start` to confirm nothing broke
